BuildingThemes/Configuration.cs

```diff
@@ -35,13 +36,13 @@ public class Theme
             [XmlArrayItem(ElementName = "Building")]
             public List<Building> buildings = new List<Building>();
 
-            public void addAll(string[] buildingNames, bool builtIn)
+            public void addAll(string[] buildingNames, string[] removedBuildings, bool builtIn)
```

Comment:
You don't need to change this. The foreach statement below will only add buildings that are not already in the list (see containsBuilding method). And the buildings which were disabled by the user are already in the list when the vanilla buildings are added.